src/app/profiles/store/selectors/contextPopin.ts

```diff
@@ -0,0 +1,9 @@
+import { ProfilesState } from '../reducers';
+import { createSelector } from 'reselect';
+
+export const getContextPopinState = (state: ProfilesState) =>
+  state.contextPopin;
+export const getContextPopinContent = createSelector(
+  [getContextPopinState],
+  content => content
```

Comment:
Shouldn’t that be `({ content }: ContextPopinState) => content` ?

Reply:
you'r right